### ISSUE TITLE
Integrity Check results in Error

### ISSUE THREAD

**Issue by voltagex007** [pure-css/pure#852]
### Description

My Firefox Developer Console tells that the sha384 Integrity has some errors, resulting in problems in accessing the css file (this means It never got the css file as it didn't works)

### Expected behavior
It should do nothing other than grab the css framework file using the unpkg link (as provided in the main site) and work properly.

### Actual behavior
The actual behavior is, using pure through unpkg results and error in FireFox Console:
`None of the “sha384” hashes in the integrity attribute match the content of the subresource.`
This is the only behavior, I can still use the normalize.css (as I can notice font family getting changed) while trying to make a menu (horizontal). It just becomes a list rather than pure css menu. Also without the **anonymous** and integrity, it works good.

### Steps to Reproduce

Just create a file called "index.html" and type this code in...

```
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Pure</title>

	<link rel="stylesheet" href="https://unpkg.com/purecss@2.0.5/build/pure-min.css" integrity="sha384-G9DpmGxRIF6tpgbrkZVcZDeIomEU22LgTguPAI739bbKytjPE/kHTK5YxjJAAEXC" crossorigin="anonymous">
	
	<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.1/build/base-min.css">

</head>
<body>

	<div class="pure-menu pure-menu-horizontal">
		<a href="#" class="pure-menu-heading pure-menu-link">MAIN</a>
		<ul class="pure-menu-list">
			<li class="pure-menu-item">
				<a href="#" class="pure-menu-link">Epic History</a>
			</li>
			<li class="pure-menu-item">
				<a href="#" class="pure-menu-link">Download</a>
			</li>
			<li class="pure-menu-item">
				<a href="#" class="pure-menu-link">Mega Docs</a>
			</li>
		</ul>
	</div>
	
</body>
</html>
```

Then open this index.html file in Firefox...

### Environment

* Pure version: 2.0.5
* Browser name and version: Firefox 87.0
* System Info: Linux, Ubuntu 18.04 LTS based Zorin OS Lite 15.3 (Updated the system yesterday, using the classic command: `sudo apt upgrade`)

> IDK whats the purpose of system info is, but still wanted to provide in case your team needs it.

Waiting for a reply, I'm going to use Pure for my few future websites as the CSS Framework.

Hopefully, I get a quick response  :)

~V

**Comment by redonkulus** [comment]
We had this [issue before](https://github.com/pure-css/pure/issues/828) and I thought I fixed it, will need to check the build again.

**Comment by redonkulus** [comment]
@voltagex007 this should be fixed in v2.0.6 now. When I [upgraded](https://github.com/pure-css/pure/commit/fa740b056442d4f43c61bac8a9b737244cfd71cf) the dependency to cssmin@4, I forgot to issue a new release because some of the output changed in cssmin. 

Please try the new hash from the website and let me know if that fixes things.

**Comment by voltagex007** [comment]
Thanks a Lot, It now works. It's always happy to see people fix it quick (but sad to see that I can't contribute somehow, which is my main goal, just contribute). Now my huge website project can be made out of Pure CSS which I felt is better than Bootstrap.

~V
